LICENSE.md

```diff
@@ -1,7 +1,7 @@
 The MIT License (MIT)
 =====================
 
-Copyright (c) 2015-2018 Vladimir Iakovlev
+Copyright (c) 2020-2023 Vladimir Iakovlev
```

Comment:
Please read [this](https://www.gnu.org/prep/maintain/maintain.html#Copyright-Notices).

Reply:
```suggestion
Copyright (c) 2015-2020 Vladimir Iakovlev
```